<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadSettings</class>
 <widget class="QDialog" name="DownloadSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>576</width>
     <height>263</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QSpinBox" name="spinBoxDownloadIndexStart"/>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="checkBoxDownloadThumbnail">
      <property name="text">
       <string>Download Thumbnail</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBoxLimitNumVideosDownloadedFromPlaylist">
      <property name="text">
       <string>Limit Number of Videos Downloaded from Playlist</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="checkBoxNormalizeFilenames">
      <property name="text">
       <string>Normalize Filenames</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBoxDownloadRangeFromPlaylist">
      <property name="text">
       <string>Download Range from Playlist</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEditMaxFilenameLength"/>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="checkBoxLimitFilenameLength">
      <property name="text">
       <string>Limit Filename Length</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="checkBoxWindowsFilenames">
      <property name="text">
       <string>Windows Filenames</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBoxDownloadLimit"/>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBoxDownloadPlaylistRandomOrder">
      <property name="text">
       <string>Download Playlist Videos in a Random Order</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBoxDownloadLinkedPlaylist">
      <property name="text">
       <string>If a Playlist is Linked, download the whole playlist</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QSpinBox" name="spinBoxDownloadIndexEnd"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DownloadSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DownloadSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
